### PR TITLE
Web Server Simply

### PR DESCRIPTION
**Pull request by AleksandraPonochovna**
HW 19 and HW 20 is here

### CODE REVIEW

File: src/main/java/factory/ProductsDaoFactory.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package factory;
+
+import dao.Dao;
+import dao.ProductsDao;
+
+public class ProductsDaoFactory {
```
Comment (Sofasmile) [inline src/main/java/factory/ProductsDaoFactory.java]: we can create one factory for two classes
for example:
https://vertex-academy.com/tutorials/ru/pattern-factory-java/

File: src/main/java/model/User.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Override
+    public String toString() {
+        return "Users{" +
+                "id=" + id +
+                ", email='" + email + '\'' +
+                ", password='" + password + '\'' +
+                '}';
+    }
+}
```
Comment (Sofasmile) [inline src/main/java/model/User.java]: a blank line

File: src/main/java/factory/ProductsDaoFactory.java (hunk 1 above)
Comment (AleksandraPonochovna) [reply src/main/java/factory/ProductsDaoFactory.java]: Yes, but German said us that we do not one Factory. 
Even today he said it again
Maybe I don't understand it rightly, I am not sure. 
And today he said that in Factory we should use with Service and i will change it

Comment (AleksandraPonochovna) [reply src/main/java/factory/ProductsDaoFactory.java]: And we should do for User UserServiceFactory, for Product - ProductServiceFactory

File: src/main/java/collector/AddingProductServlet.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/addingproduct")
+public class AddingProductServlet extends HttpServlet {
+    private Dao<Product> productDao = ProductsDaoFactory.getProductDao();
+    private Product product;
+    private Long id = 1L;
```
[the remark this answers is not in the document]
Comment (AleksandraPonochovna) [reply src/main/java/collector/AddingProductServlet.java]: Ok, i will change it

File: src/main/java/collector/RegistrationServlet.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Objects;
+
+@WebServlet(value = "/register")
+public class RegistrationServlet extends HttpServlet {
+    private Dao<User> userDao = UsersDaoFactory.getUserDao();
+    private User user;
+    private Long id = 1L;
```
[the remark this answers is not in the document]
Comment (AleksandraPonochovna) [reply src/main/java/collector/RegistrationServlet.java]: Ok

File: src/main/java/dao/ProductsDao.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+import model.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductsDao implements Dao<Product> {
+    private List<Product> products = new ArrayList<>();
+
+    public List<Product> getProducts() {
```
[the remark this answers is not in the document]
Comment (AleksandraPonochovna) [reply src/main/java/dao/ProductsDao.java]: I think that it's in the future, 'cause our project is more and more.
But I can delete it, it's not problem

File: src/main/java/dao/UsersDao.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UsersDao implements Dao<User> {
+    private List<User> users = new ArrayList<>();
+
+    public List<User> getUsers() {
```
[the remark this answers is not in the document]
Comment (AleksandraPonochovna) [reply src/main/java/dao/UsersDao.java]: Ok

File: src/main/java/database/Database.java [hunk 7]
```diff
@@ -0,0 +1,28 @@
+package database;
+
+import model.Product;
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Database {
+    private static List<Product> products = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (AleksandraPonochovna) [reply src/main/java/database/Database.java]: I delete all the methods here

**Pull request closed without merging**